index.html

```diff
@@ -255,5 +335,6 @@ <h2><a href="#enhancement_requests">Enhancement Requests</a></h2>
 <script>
 createtoc(3)
 </script>
+<script type="text/javascript" src="createGitHubIssue.js"></script>
```

Comment:
```suggestion
<script src="createGitHubIssue.js"></script>
```

To avoid warnings in HTML validator.